docs/reference/scripting/security.asciidoc

```diff
@@ -61,3 +67,42 @@ For example, to allow scripts to run only in `scoring` and `update` contexts:
 ----
 script.allowed_contexts: score, update
 ----
+
+[[allowed-script-in-aggs-settings]]
+[discrete]
+=== Allowed scripts in scripted metrics aggregations
+
+By default, all scripts are permitted in
+<<search-aggregations-metrics-scripted-metric-aggregation,scripted metrics aggregations>>.
+To restrict the set of allowed scripts, set
+<<search-settings-only-allowed-scripts,`search.aggs.only_allowed_metric_scripts`>>
+to `true` and provide the allowed scripts using
+<<search-settings-allowed-inline-scripts,`search.aggs.allowed_inline_metric_scripts`>>
+and
+<<search-settings-allowed-stored-scripts,`search.aggs.allowed_stored_metric_scripts`>>.
+
```

Comment:
```suggestion

To explicitly disallow scripts, set the corresponding script list 
(`search.aggs.allowed_inline_metric_scripts` or 
`search.aggs.allowed_stored_metric_scripts`) to an empty array. This 
configuration ensures that no scripts of that type are permitted.
```

This suggestion based on the examples where this fact is implicit, and just makes it explicit in advance. WDYT?

Reply:
Or because it defaults to empty list the examples are just illustrative?

Reply:
It's just illustrative, the empty list could also be omitted.

I'm going to add your suggestion, but would mention that just omitting works, too?